examples/imagenet/README.md

```diff
@@ -46,7 +46,7 @@ Configuration flag is defined using
 follows:
 
 ```shell
-python main.py --workdir=./imagenet_default --config.num_epochs=100
+python main.py --workdir=./imagenet_default --config.num_epochs=100 --config=configs/default.py
```

Comment:
Nit: we usually switch the order : `--config=config/default.py --config.num_epochs=100`

Reply:
tweaked!